stdlib/stdlib.mli

```diff
@@ -687,6 +687,10 @@ external ignore : 'a -> unit = "%ignore"
    compiler warning; writing [ignore(f x)] instead
    avoids the warning. *)
 
+val null : < .. >
```

Comment:
Missing docstrings.

Reply:
Moreover, all these new values should go to a new module `Stdlib.Null`, and `Stdlib.Null.Infix` for `(//)`.